io/src/librealsense/librealsense_device_manager.cpp

```diff
@@ -120,7 +119,7 @@ void
 pcl::io::librealsense::LibRealSenseDeviceManager::releaseDevice (const std::string& sn)
 {
   boost::mutex::scoped_lock lock (mutex_);
-  pcl::io::librealsense::LibRealSenseDeviceManager::stopDevice (sn);
+  if (!thread_close_flag_[sn]) pcl::io::librealsense::LibRealSenseDeviceManager::stopDevice (sn);
```

Comment:
Yeah, you are right, I forgot to check this, it is unnecessary to stop again in the  `LibRealSenseDeviceManager`,  I will remove stop in the `LibRealSenseDeviceManager`.

The purpose of `LibRealSenseDeviceManager::releaseDevice` is indeed just to remove the mapping, but the purpose of the map is to check if the device is captured, if the device is already in the map, the other grabber cannot use this device.